src/state/data-model-list/selectors.js

```diff
@@ -0,0 +1,13 @@
+import { createSelector } from 'reselect';
+
+export const getDataModels = state => state.dataModelList;
+
+export const getListDataModels = createSelector(
+  [getDataModels],
+  dataModelList => dataModelList.list,
+);
+
+export const getListDataModelsPaged = createSelector(
+  [getDataModels],
+  dataModelList => dataModelList.pagedList,
```

Comment:
The "paged" list should not be saved into the state, it should be calculated using the list state and the paginator state